README.md

```diff
@@ -27,6 +28,7 @@ bash <(curl -s https://install.aeternity.io/install.sh) --no-prompt [version]
 ##### Additional parameters
 
 - `TARGET_DIR` - To override the path where aeternity node will be installed (default: `$HOME/aeternity/node`)
+- `SNAPSHOT_DIR` - To override the path where the aeternity snapshot can be downloaded (default: `$TARGET_DIR/../maindb`)
```

Comment:
don't forget to update this, once the default path is changed.

Reply:
c8bad5e